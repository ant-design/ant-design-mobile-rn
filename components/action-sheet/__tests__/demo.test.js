import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('action-sheet')
