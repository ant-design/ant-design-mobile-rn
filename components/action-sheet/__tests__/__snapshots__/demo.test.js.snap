// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/action-sheet/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 30,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "padding": 8,
        },
      ]
    }
  >
    <
      activeOpacity={0.4}
      disabled={false}
      onHideUnderlay={[Function]}
      onPress={[Function]}
      onPressIn={[Function]}
      onPressOut={[Function]}
      onShowUnderlay={[Function]}
      pressIn={false}
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "borderWidth": 1,
            "justifyContent": "center",
          },
          Object {
            "height": 47,
            "paddingLeft": 15,
            "paddingRight": 15,
          },
          Object {
            "backgroundColor": "#ffffff",
            "borderColor": "#dddddd",
          },
          false,
          false,
          undefined,
          undefined,
        ]
      }
      underlayColor="#dddddd"
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontSize": 18,
              },
              Object {
                "color": "#000000",
              },
              false,
              false,
            ]
          }
        >
          showActionSheet
        </Text>
      </View>
    </>
  </View>
  <Text
    style={
      Array [
        Object {
          "padding": 8,
        },
      ]
    }
  >
    clicked button: 
    none
  </Text>
  <View
    style={
      Array [
        Object {
          "padding": 8,
        },
      ]
    }
  >
    <
      activeOpacity={0.4}
      disabled={false}
      onHideUnderlay={[Function]}
      onPress={[Function]}
      onPressIn={[Function]}
      onPressOut={[Function]}
      onShowUnderlay={[Function]}
      pressIn={false}
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderRadius": 5,
            "borderWidth": 1,
            "justifyContent": "center",
          },
          Object {
            "height": 47,
            "paddingLeft": 15,
            "paddingRight": 15,
          },
          Object {
            "backgroundColor": "#ffffff",
            "borderColor": "#dddddd",
          },
          false,
          false,
          undefined,
          undefined,
        ]
      }
      underlayColor="#dddddd"
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontSize": 18,
              },
              Object {
                "color": "#000000",
              },
              false,
              false,
            ]
          }
        >
          showShareActionSheet
        </Text>
      </View>
    </>
  </View>
  <Text
    style={
      Array [
        Object {
          "padding": 8,
        },
      ]
    }
  >
    
  </Text>
</View>
`;
